src/backend/_scss/application.scss

```diff
@@ -522,39 +522,63 @@ a {
   min-height: 255px;
   position: absolute;
   transition: all 0.5s ease;
+  background-color:white;
```

Comment:
Everything has been sorted out @aayusharora , I don't have any more issues in scss linting. Check the latest status please.